src/sinol_make/helpers/compiler.py

```diff
@@ -25,7 +25,7 @@ def get_c_compiler_path():
 		else:
 			return 'gcc'
 	elif sys.platform == 'darwin':
-		for i in range(9, 13):
+		for i in range(9, 11):
```

Comment:
that one is `range`, but the below one is `[]`

Reply:
Sorry, I forgot about it